### ISSUE TITLE
Add a chan_sip to the configuration

### ISSUE THREAD

**Issue by khdevel** [dougbtv/docker-asterisk#40]
Hi,

I like your Dockerfile for the asterisk. I run it and it seems to work... expect that it does not have a sip port open. I saw that there is a miss of the _chan_sip_ in the configuration. I checked also the _make menuconfig_ and it is not set.

Of course I tried to set it by myself but without the success. The command:
```
RUN menuselect/menuselect --enable chan_sip menuselect.makeopts
```
did not work for me, so I made a changes by some _sed_ like:
```
RUN sed -i -e 's/\ chan_sip//' menuselect.makeopts
RUN sed -i -e 's/G711_NEW_ALGORITHM/res_crypto res_http_websocket G711_NEW_ALGORITHM/' menuselect.makeopts
RUN sed -i -e 's/res_fax_spandsp\ res_http_websocket/res_fax_spandsp/' menuselect.makeopts
```
but it still does not work - the port 5060 is not open like it is with IAX in your default config.

Could you please help me and update the Dockerfile for the SIP protocol too.

EDITED
-----------------------

I run my build again and this:
```
RUN sed -i -e 's/\ chan_sip//' menuselect.makeopts
RUN sed -i -e 's/G711_NEW_ALGORITHM/res_crypto res_http_websocket G711_NEW_ALGORITHM/' menuselect.makeopts
RUN sed -i -e 's/res_fax_spandsp\ res_http_websocket/res_fax_spandsp/' menuselect.makeopts
```
worked for me! The sip works fine and the port is open.
Do you think it is a good moment for some pull request? Or maybe you know how to change above _sed_ on some better way?

Thank you!

**Comment by dougbtv** [comment]
Hrmmm, thanks for the issue submission + taking a look. I just added a
Travis build for the project a week ago and have tests where a SIP call is
made between two boxes so you might want to take a look at the .travis.yml
file and see how it's run, in the meanwhile.

I'll take another look this coming week, I'm on the road this weekend, so I
might not be able to give it a lot of attention until then.

I'll update soon!

**Comment by dougbtv** [comment]
I'm still mobile, but check out the latest build:
https://travis-ci.org/dougbtv/docker-asterisk

`core show modules` in the build shows chan_sip -- what's your docker run
command look like and what does your core show modules look like?

**Comment by dougbtv** [comment]
@khdevel -- glad to see that worked! definitely submit a PR, happy to take a look with the changes, should be a-OK as it is. Thanks for digging through it!

**Comment by dougbtv** [comment]
I'm going to close this out for now. If you've got another recommendation, I'm happy to reopen it and look further however I currently believe chan_sip to be enabled.

**Comment by khdevel** [comment]
Hi,
Sorry for no reply from my side, something else disturbed my mind and I did not follow this topic... I will test it this week I hope and I believe that it works as you wrote above :) 

Thank you very much!

**Comment by dougbtv** [comment]
Thanks @khdevel -- I'll open the issue back up and happy to collab if you're not getting the results you expect for sure.

**Comment by dougbtv** [comment]
@khdevel -- I was working on the asterisk13 image and I noticed that it didn't have chan_sip! Just pjsip!

So, I fixed that merged in with #49  . Now...... now you should have some luck! If I don't hear from you for a couple days, I'll close 'er out, I think I finally found what you had tried to show me, my bad for not picking up on it sooner.

**Comment by khdevel** [comment]
Hi,

I have tested you fix and it seems to work fine. I did not use your code from GitHub but I used the image from the Docker's repository _dougbtv/asterisk13_.
My host was Centos7 and after I run the container I could register my two SIP phones and make a call between them. No modifications were needed except those I had to make to register and make a call (sip.conf and extensions.conf).

Thank you very much!
